### PR TITLE
Switched products.visible for products.visibility

### PR DESCRIPTION
**Pull request by matthanley**
- We have an existing pattern for using `visibility: public` instead of `visible: true|false`
- We no-op the existing migration and roll forward so that we don't have to manually revert db changes

### CODE REVIEW

File: core/server/data/migrations/versions/4.38/2022-03-01-08-46-add-visibility-to-tiers.js [hunk 1]
```diff
@@ -5,3 +5,7 @@ module.exports = createAddColumnMigration('products', 'visible', {
     nullable: false,
     defaultTo: false
 });
+
+module.exports.up = async () => {
```
Comment (allouis) [inline core/server/data/migrations/versions/4.38/2022-03-01-08-46-add-visibility-to-tiers.js:9]: Why is this file not being deleted?

**Review by allouis — COMMENTED** [review]
Changes look good - I don't understand why we're keeping the unreleased migrations though - isn't it simpler to completely remove them?

Is this to stop issues with local development on `main`?

**Review by allouis — APPROVED** [review]
Either way this looks good though!

**Review by daniellockyer — APPROVED** [review]

**Comment by daniellockyer** [comment]
> Is this to stop issues with local development on main?

@allouis Yeah, that, and we decided to always move forwards with migrations once they've landed in main. We have some internal sites that are continuously deployed without resetting their DB, and deleting the migration will brick their DB because knex-migrator won't like the extra row

**Comment by allouis** [comment]
Cool cool cool cool cool 👌

**Pull request merged**